Readme.md

```diff
@@ -34,7 +34,10 @@ Here's a quick public tour of Open Library to get your familiar with the service
 
 ## Installation
 
-The development environment can be set up using the [Docker Instructions](https://github.com/internetarchive/openlibrary/blob/master/docker/README.md). You can also watch the [video tutorial](https://archive.org/embed/openlibrary-developer-docs/openlibrary-docker-set-up.mp4) for a more detailed explanation.
+Run `docker-compose up` and visit http://localhost:8080
```

Comment:
Do you think that we should explicitly mention that Docker Compose needs to be installed before running this command?

Reply:
I think that's a fair question. I think I've seen plenty of projects not mention it and plenty mention it. I plan to clean up the docs further, especially as we add gitpod. I like to keep the main readme simple and link out to the more detailed docs but that's just me.

Reply:
Keeping the project readme simple seems reasonable to me.